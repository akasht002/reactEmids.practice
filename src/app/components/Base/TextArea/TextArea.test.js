import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import { TextArea } from './index';

Enzyme.configure({ adapter: new Adapter() })

describe('SelectBox', () => {
  it('should return correct component', () => {
    const wrapper = shallow(
      <TextArea id='1' placeholder='Write Some Thing' shrink={99} rows='5' className='textarea'></TextArea>,
    )
    expect(wrapper).toBeDefined();
  })
});





