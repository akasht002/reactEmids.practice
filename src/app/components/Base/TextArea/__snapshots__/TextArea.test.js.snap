// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea Component Jest 1`] = `
<div
  className="form-group"
>
  <label />
   
  <i />
  <textarea
    className="textarea"
    placeholder="Write Some Thing"
    rows="5"
  />
</div>
`;
